aws-lambda/README.md

```diff
@@ -0,0 +1,101 @@
+# ![](./img/integration_awslambda.png) Amazon Lambda
+
+- [Description](#description)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+Use SignalFx to monitor Lambda via [Amazon CloudWatch](https://github.com/signalfx/integrations/tree/master/aws)[](sfx_link:aws).
```

Comment:
I propose changing this info to be closer to what I have in the docs (TBD which comes first, CW or wrapper). 

If you have enabled the SignalFx Amazon Web Services integration and are syncing Cloudwatch metrics, metrics related to Lambda functions will automatically be available to view in SignalFx.

You can also use one of our wrappers to monitor your functions. If you use a wrapper, in addition to viewing the standard metrics our wrappers make available, you can send in custom metrics to monitor apps running inside the Lambda. Wrappers also provide real-time data to SignalFx.